packages/interface/src/components/complementary-area/index.js

```diff
@@ -134,22 +136,33 @@ function ComplementaryArea( {
 
 	return (
 		<>
-			{ isPinned && isPinnable && (
+			{ isPinnable && (
 				<PinnedItems scope={ scope }>
-					<ComplementaryAreaToggle
-						scope={ scope }
-						identifier={ identifier }
-						isPressed={
-							isActive && ( ! showIconLabels || isLarge )
-						}
-						aria-expanded={ isActive }
-						label={ title }
-						icon={ showIconLabels ? check : icon }
-						showTooltip={ ! showIconLabels }
-						isTertiary={ showIconLabels }
-					/>
+					{ isPinned && (
+						<ComplementaryAreaToggle
+							scope={ scope }
+							identifier={ identifier }
+							isPressed={
+								isActive && ( ! showIconLabels || isLarge )
+							}
+							aria-expanded={ isActive }
+							label={ title }
+							icon={ showIconLabels ? check : icon }
+							showTooltip={ ! showIconLabels }
+							isTertiary={ showIconLabels }
+						/>
+					) }
 				</PinnedItems>
 			) }
+			{ name && isPinnable && (
```

Comment:
I found a prop that is mandatory and unique for the `PluginSidebar` implementation. Would it work?

Reply:
is the name the prop?

Reply:
Yes, it's part of the `PluginSidebar` API. In other places `identifier` is used for some reasons.